hw_diag/templates/template_hyper.html

```diff
@@ -240,34 +239,66 @@
         <!-- Sidenav scroll fix -->
         <script>
             $(document).ready(function() {
+                var navTogglerClicked = false;
+              
                 // Check initial value of data-sidenav-size
                 checkSidenavSize();
-              
+                  
                 // Create a new MutationObserver to watch for changes in the html element
                 var observer = new MutationObserver(function(mutations) {
                   mutations.forEach(function(mutation) {
                     if (mutation.attributeName === 'data-sidenav-size') {
                       checkSidenavSize();
-                      console.log("sidenav state changed");
+                      //console.log("sidenav state changed");
                     }
                   });
                 });
-              
+                  
                 // Attach the observer to the html element
                 observer.observe(document.querySelector('html'), {attributes: true});
-              
+                
+                // Function to check sidenav size and set margin-left property accordingly
                 function checkSidenavSize() {
                   var sidenavSize = $('html').attr('data-sidenav-size');
-                
+              
                   if (sidenavSize === 'full') {
                     $('#navbar-toggler').css('margin-left', '0');
-                    console.log("sidenav closed");
+                    //console.log("sidenav closed");
```

Comment:
Why are we leaving commented code in?

Reply:
I don't have that on my copy. Either way it should resolve it when this is reviewed.